addons/website/static/src/builder/plugins/floating_blocks_option_plugin.js

```diff
@@ -0,0 +1,126 @@
+            withSequence(after(DEVICE_VISIBILITY), {
+                OptionComponent: FloatingBlocksOption,
+                selector: ".s_floating_blocks",
+            }),
+        ],
+        builder_actions: {
+            floatingBlocksRoundness: {
+                getValue: ({ editingElement }) => {
+                    for (let x = 0; x <= 5; x++) {
+                        if (editingElement.classList.contains(`rounded-${x}`)) {
+                            return x;
+                        }
+                    }
+                    return 0;
+                },
+                apply: ({ editingElement, value }) => {
+                    for (let x = 0; x <= 5; x++) {
+                        editingElement.classList.remove(`rounded-${x}`);
+                    }
+                    editingElement.classList.add(`rounded-${value}`);
+                },
+            },
+            addCard: {
+                apply: ({ editingElement: el }) => {
+                    const newCardEl = renderToElement("website.s_floating_blocks.new_card");
+                    const wrapperEl = el.querySelector(".s_floating_blocks_wrapper");
+                    wrapperEl.appendChild(newCardEl);
+                    newCardEl.scrollIntoView({ behavior: "smooth", block: "center" });
+                    this.validateBoxesNumber(el);
+                    this.dependencies.edit_interaction.restartInteractions(el);
+                },
+            },
+        },
+        clean_for_save_handlers: this.cleanForSave.bind(this),
+        normalize_handlers: this.normalize.bind(this),
+    };
+
+    cleanForSave({ root }) {
+        for (const el of root.querySelectorAll(".s_floating_blocks_wrapper")) {
+            const boxesEls = [...el.querySelectorAll(".s_floating_blocks_block")];
+            const alertEl = el.querySelector(".s_floating_blocks_alert_empty");
+            if (boxesEls.length > 0) {
+                alertEl?.remove();
+            } else {
+                // Special case: by injecting the "No cards" alert ('alertEl'), we
+                // prevent the automatic snippet removal during edition. Still, if
+                // the user intentionally "saves" the snippet empty, we'll emulate
+                // the original editor behavior by removing it here.
+                // See also FloatingBlocksOption.setup().
+                el.closest(".s_floating_blocks").remove();
+            }
+        }
+    }
+
+    normalize(el) {
+        // Counts the blocks and restarts the interaction every time that a
+        // block is added, removed or moved. Note that interaction must
+        // restart on block movement to get the correct visual effect.
+        const wrapperEl = el.closest(".s_floating_blocks_wrapper");
+        if (wrapperEl) {
+            this.validateBoxesNumber(wrapperEl);
+            this.dependencies.edit_interaction.restartInteractions(
+                wrapperEl.closest(".s_floating_blocks")
+            );
+        }
```

Comment:
It s not done by default now ?

Reply:
You should check shouldStop. It s away to configurate when you need to restart or not your interaction. By default, it will check if an data attribute change
Exemple: community/addons/website/static/src/interactions/full_screen_height.edit.js